spec/std/file_spec.cr

```diff
@@ -958,6 +958,40 @@ describe "File" do
         end
       end
     end
+
+    it "#flock_shared non-blocking" do
```

Comment:
It's kinda confusing that this is when `blocking = true`, but it's "non-blocking". Also, this is not yet implemented in win32.

Perhaps
```suggestion
    pending_win32 "#flock_shared yielding when blocking" do
```